package.json

```diff
@@ -302,6 +305,7 @@
     "calculate-size": "1.1.1",
     "centrifuge": "3.1.0",
     "classnames": "2.3.2",
+    "codeowners": "^5.1.1",
```

Comment:
this should probably just be a `devDep` i think

Reply:
fix ta thanks :)